addon/mixins/routes/application.js

```diff
@@ -31,6 +31,13 @@ export default Ember.Mixin.create({
         outlet: outlet,
         parentView: parentView
       });
+
+      // Check if another modal exists then give it focus
+      Ember.run.later(function() {
```

Comment:
What's the benefit of using `Ember.run.later` without a duration as the third argument? If it's absolutely necessary to use the run loop perhaps `Ember.run(function()` or `Ember.run.next(function()` would suffice.